spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/LoadBalancerServiceProperties.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2012-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.client.loadbalancer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * A {@link ConfigurationProperties} bean for client LoadBalancer.
+ *
+ * @author Andrii Bohutskyi
+ */
+@ConfigurationProperties("spring.cloud.loadbalancer.client")
+public class LoadBalancerServiceProperties {
+
+	private Map<String, LoadBalancerProperties> services = new HashMap<>();
```

Comment:
Could you modify the implementation, so that the property is  `spring.cloud.loadbalancer.client.foo1.healthcheck...` as opposed to `spring.cloud.loadbalancer.client.services.foo1.healthcheck...`?

Reply:
I agree